specs/OracleQueries.yml

```diff
@@ -468,6 +485,36 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/activeOfficerDetails'
+    managingOfficerData:
+      type: object
+      required:
+        - items_per_page
+        - start_index
+        - total_results
+        - items
+      properties:
+        items_per_page:
+          type: integer
+          format: int64
+          description: Number of managing officers per page returned in this list
+          readOnly: true
+          example: 10
+        start_index:
+          type: integer
+          format: int64
+          description: The offset into the entire list that this page starts at. Zero indexed
+          readOnly: true
+          example: 0
+        total_results:
+          type: integer
+          format: int64
+          description: The total number of managing officers in the list
+          readOnly: true
+          example: 0
+        items:
+          type: array
+          items:
+            $ref: '#/components/schemas/managingOfficerData'
```

Comment:
This should point to a schema object with the required fields- see above referenced (line 487) activeOfficerDetails as an example

Reply:
fixed